README.md

```diff
@@ -129,7 +124,7 @@ x = 100;
 ```
 
 In the example above, `x` is annotated with the type `string`.
-Tools such as TypeScript can utilize that type, and might choose to error on the statement `x = 100`;
+Tools doing static type analysis can utilize that type, and might choose to error on the statement `x = 100`;
```

Comment:
Static analysis tools can utilize that type....

(active over passive voice)

Reply:
(I don't think that's active vs passive voice, an adverbial phrase that could be rewritten "tools, such as typescript, can ....")